frontend/optmath.lua

```diff
@@ -73,4 +73,8 @@ function Math.tmax(tab, func)
     return tmin_max(tab, func, "max")
 end
 
+function Math.roundPercent(percent)
```

Comment:
Please put it above by the other round functions instead of randomly at the bottom. ^_^